app/service/file_svc.py

```diff
@@ -87,6 +87,24 @@ def __init__(self, plugins, exfil_dir):
             return name, xor_file(file_name)
         raise FileNotFoundError
 
+    async def save_file(self, name, content, xored=False):
+        """
+        Save a (payload) file to the stockpile
+        :param name: filename
+        :param content: content to save
+        :param xored: whether or not to xor the contents when saving
+        :return: full path of the saved file
+        """
+        filebase='plugins/stockpile/payloads/'
+        f_content = content
+        filename = str(os.path.join("dummy",name).split(os.path.sep)[-1])
```

Comment:
Make all quotes single quotes

Reply:
is "dummy" supposed to be `filebase`?

Reply:
Or is that a directory traversal prevention?

Reply:
It's a trick to prevent directory traversal. I'll rename dummy, though, since literally a single character could accomplish the same thing.